content/en/blog/falco-0-29-0.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Falco 0.29.0
+date: 2021-06-21
+author: Massimiliano Giovagnoli
+slug: falco-0-29-0
+---
+
+Today we announce the summer release of Falco 0.29.0 🌱
+
+This is a minor version which brings new features and fixes!
+
+## Novelties 🆕
+
+Let's now review some of the new things Falco 0.29.0 brings.
+
+### New libraries repostiory!
+
+First of all, as per [this proposal](https://github.com/falcosecurity/falco/blob/master/proposals/20210119-libraries-contribution.md), the repo [falcosecurity/libs](https://github.com/falcosecurity/libs) is the new home for [`libscap`](https://github.com/falcosecurity/libs/tree/master/userspace/libscap), [`libsinsp`](https://github.com/falcosecurity/libs/tree/master/userspace/libsinsp), and the Falco [drivers](https://github.com/falcosecurity/libs/tree/master/driver).
+
+With this release the building system is updated to point to the [new location](https://download.falco.org/?prefix=driver/13ec67ebd23417273275296813066e07cb85bc91/) and also the [driver version](https://download.falco.org/?prefix=driver/13ec67ebd23417273275296813066e07cb85bc91/) is updated.
+
+### New libs version!
+
+The update to the [drivers](https://github.com/falcosecurity/libs/tree/master/driver) version [17f5d](https://github.com/falcosecurity/libs) brings new features/fixes:
+- [support](https://github.com/falcosecurity/libs/pull/50) to trace the [userfaultd](https://www.kernel.org/doc/html/latest/admin-guide/mm/userfaultfd.html) system call
+- [improvement](https://github.com/falcosecurity/libs/pull/32) in [`libsinsp`](https://github.com/falcosecurity/libs/tree/master/userspace/libsinsp) on the way how pod resources limits and pod IP are gathered from container runtime
+- [improvement](https://github.com/falcosecurity/libs/pull/15) in [`libsinsp`](https://github.com/falcosecurity/libs/tree/master/userspace/libsinsp) on pod metadata and namespace retrievial for large cluster scenarios, by getting them directly from container labels and bypassing the API server
```

Comment:
```suggestion
- [improvement](https://github.com/falcosecurity/libs/pull/15) in [`libsinsp`](https://github.com/falcosecurity/libs/tree/master/userspace/libsinsp) on pod metadata and namespace retrieval for large cluster scenarios, by getting them directly from container labels which is more efficient and use the K8s API server as a fallback
```

Reply:
IMO, it worth also mentioning:
- https://github.com/falcosecurity/libs/pull/22
- https://github.com/falcosecurity/libs/pull/42

Reply:
agree